src/main/java/productDependency/CreateOrderPeriodPage.java

```diff
@@ -0,0 +1,234 @@
+
+	/**
+	 * Method to click on Add New Button.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickAddNewButton() throws IOException {
+		CreateOrderPeriodPage sp = new CreateOrderPeriodPage(driver);
+		log.info("Click on Add New");
+		Assert.assertTrue(clickAddNewButton.isDisplayed());
+		clickAddNewButton.click();
+		JavaScriptExec.sleep();
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//input[@name='description']")
+	private WebElement enterDescription;
+
+	/**
+	 * Method to enter Description.
+	 * 
+	 * @throws IOException
+	 */
+	public void enterDescription() throws IOException {
+		CreateOrderPeriodPage sp = new CreateOrderPeriodPage(driver);
+		log.info("Verifying the Description is available or not");
+		Assert.assertTrue(enterDescription.isDisplayed());
+		enterDescription.sendKeys(BasePage.getCellData(xlsxName, sheetName, 3, 0));
+
+	}
+
+	private WebElement selectUnit;
+
+	/**
+	 * Method to select Unit.
+	 * 
+	 * @throws IOException
+	 */
+	public void selectUnit() throws IOException {
+		CreateOrderPeriodPage sp = new CreateOrderPeriodPage(driver);
+		WebElement Unitelement = driver.findElement(By.xpath("//select[@name='periodUnitId']"));
+		Select se = new Select(Unitelement);
+		se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 4, 0));
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//input[@name='value']")
+	private WebElement enterValue;
+
+	/**
+	 * Method to enter Value.
+	 * 
+	 * @throws IOException
+	 */
+	public void enterValue() throws IOException {
+		CreateOrderPeriodPage sp = new CreateOrderPeriodPage(driver);
+		log.info("Verifying the Description is available or not");
+		Assert.assertTrue(enterValue.isDisplayed());
+		enterValue.sendKeys(BasePage.getCellData(xlsxName, sheetName, 5, 0));
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//a[@class='submit save']//*[text()='Save Changes']")
+	private WebElement clickSaveChangesButton;
+
+	/**
+	 * Method to click on Save Changes Button.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickSaveChangesButton() throws IOException {
+		CreateOrderPeriodPage sp = new CreateOrderPeriodPage(driver);
+		log.info("Click on Save Changes Button");
+		Assert.assertTrue(clickSaveChangesButton.isDisplayed());
+		clickSaveChangesButton.click();
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//div[@class='msg-box successfully']//*[text()='Done']")
+	private WebElement verifyConfirmationMsg;
+
+	/**
+	 * Method to verify Order Period is created Successfully.
+	 * 
+	 * @throws IOException
+	 */
+	public void verifyConfirmationMsg() throws IOException {
+		CreateOrderPeriodPage sp = new CreateOrderPeriodPage(driver);
+		log.info("Verifying if Order Period is created Successfully or not");
+		JavaScriptExec.sleep();
+		Assert.assertTrue(verifyConfirmationMsg.isDisplayed(),
+				"Assert Failed as its unable to search text in Logged in Page");
+	}
+
+	public void navigateBottom() {
+		JavaScriptExec.scrolltoBottomofPage(driver);
+		JavaScriptExec.sleep();
+	}
+
```

Comment:
@mayankchhajer remove extra lines from all the method and select the class by using Ctrl+A and Format the code by using Ctrl+Shift+F , do this in all the classes

Reply:
Done